python/lsst/meas/extensions/shapeHSM/_hsm_moments.py

```diff
@@ -267,13 +267,13 @@ def measure(self, record, exposure):
         bounds = galsim.bounds.BoundsI(xmin, xmax, ymin, ymax)
 
         # Get the `lsst.meas.base` mask for bad pixels.
-        badpix = exposure[bbox].mask.array.copy()
+        badpix = exposure.mask[bbox].array.copy()
         bitValue = exposure.mask.getPlaneBitMask(self.config.badMaskPlanes)
         badpix &= bitValue
 
         # Extract the numpy array underlying the image within the bounding box
         # of the source.
-        imageArray = exposure[bbox].getImage().array
+        imageArray = exposure.getImage()[bbox].array
```

Comment:
I think this can be `exposure.image[bbox].array` if we're making things more modern.